themes/finna2/js/finna-layout.js

```diff
@@ -463,6 +463,14 @@ finna.layout = (function finnaLayout() {
     }
   }
 
+  function initResultsLoadFocus () {
+    VuFind.listen('vf-results-load', () => {
+      if (document.getElementById("results-heading")) {
+        document.getElementById("results-heading").focus();
```

Comment:
Mieluummin näin:
```
const heading = document.getElementById("results-heading");
if (heading) {
    heading.focus();
}

```
Sillä välttää toiston ja riskin, että toinen selectori jää päivittämättä joskus tulevaisuudessa.